package.json

```diff
@@ -22,9 +22,11 @@
     "url": "https://github.com/karlhorky/react-malarkey/issues"
   },
   "homepage": "https://github.com/karlhorky/react-malarkey#readme",
+  "dependencies": {
+    "malarkey": "^1.3.3"
```

Comment:
Well, Malarkey is Frontend and React is Frontend. Chances are that if you are using React, you want to use this module and not another Malarkey instance as well. I think peerDependencies are great for things that are obvious (like React), but you would probably forget to install malarkey along with the module and get a warning. At least it happened to me :)